### ISSUE TITLE
Empty instanced worlds

### ISSUE THREAD

**Issue by ArVdC** [cybertiger/Bukkit-Instances#75]
Hi Cyber,
I've a big issue since the 1.9.4 Spigot's update.
I get the same result either with Instances 1.24 and 1.25 : I get an empty map instead the expected instanced world.
However the log file doesn't show any error when I enter in an instance.
[latest.txt](https://github.com/cybertiger/Bukkit-Instances/files/288414/latest.txt)
If you want to see the result by yourself, my servername is _vdc.serveminacraft.net_.
Thank you so much for your concern.

**Comment by Scratch741** [comment]
Have you solved this? I'm getting void worlds on 1.9.4 too, even with instances 1.26

**Comment by ArVdC** [comment]
No, I never solved this. Indeed I don't use this plugin anymore.
I actually use WorldEdit's and WorldGuard's commands to restore parts of the map :
```
/rg select [region_name]
/snap use latest
//restore
```
But that is far from perfect solution ...
Check the doc at http://wiki.sk89q.com/wiki/WorldEdit/Snapshots

**Comment by Scratch741** [comment]
Thats a kind of a replacement... I stood on 1.9.2, just because of instances. Thanks for answer.
